cmd/emailreport/README.md

```diff
@@ -10,7 +10,30 @@ file can be specified by command line or it is installed in
 WARNING: The password for the SMTP authentication will be in TOML
 configuration text plane file.
 
-# Usage
+## Options
+`emailreport` takes a number of options.
+
+### Add comments/suggestions in the message body
```

Comment:
I suggest using bullets rather than third level headings for this list of options. Ensure you also indent the code block so it remains clear which bullet they belong to.

Reply:
oh, sounds good, however I was following the convention of other tools in this project:
https://github.com/clearcontainers/tests/tree/master/cmd/checkmetrics

what do you think? :)

Reply:
Alright, lets keep things consistent.